WooCommerce/Classes/Yosemite/DefaultStoresManager.swift

```diff
@@ -318,6 +312,14 @@ private extension DefaultStoresManager {
         }
         dispatch(productSettingsAction)
 
+        group.enter()
+        synchronizeSitePlan { result in
```

Comment:
I see that the method `synchronizeSitePlan` is checking for the existence of `sessionManager.defaultSite?.siteID`. In this case we already have a `siteID` to work with, so I think that check is redundant - how about moving the content of the method out here like so:
```Swift
        group.enter()
        let sitePlanAction = AccountAction.synchronizeSitePlan(siteID: siteID) { result in
            if case let .failure(error) = result {
                errors.append(error)
            }
            group.leave()
        }
        dispatch(sitePlanAction)
```
We can then remove the method and the code will be easier to read I think.